.github/workflows/nightly.yml

```
@@ -0,0 +1,50 @@
+
+name: Nightly
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  nightly:
+    concurrency:
+      group: nightly
+      cancel-in-progress: true
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java-version: [11]
+      fail-fast: true
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} - Java ${{ matrix.java-version }} - Maven
+    steps:
+      - name: Support long paths
+        if: ${{ matrix.os == 'windows-latest' }}
+        uses: kiegroup/kogito-pipelines/.ci/actions/long-paths@main
+      - name: Java and Maven Setup
+        uses: kiegroup/kogito-pipelines/.ci/actions/maven@main
+        with:
+          java-version: ${{ matrix.java-version }}
+          maven-version: ${{ matrix.maven-version }}
+          cache-key-prefix: ${{ runner.os }}-${{ matrix.java-version }}-maven${{ matrix.maven-version }}
+      - uses: actions/checkout@v2      
+      - name: Build with Maven
+        run: mvn -B clean install --file pom.xml dependency:tree -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
+      - name: Set failure message
+        if: ${{ failure() }}
+        run: echo "zulip_message='The content of the message. Maximum message size of 10000 bytes'" >> $GITHUB_ENV  
```

Is it possible to also include a link to the failed job run?
```suggestion
        run: echo "zulip_message='The optaplanner-quickstarts stable branch has failed.'" >> $GITHUB_ENV  
```

---

```suggestion
        run: echo "zulip_message='The ${{github.repository}} ${GITHUB_REF#refs/heads/} branch has failed.'" >> $GITHUB_ENV  
```